forgerock-core/src/main/java/org/forgerock/android/auth/CryptoKey.kt

```diff
@@ -30,7 +30,7 @@ class CryptoKey(private var keyId: String) {
     //For hashing the keyId
     private val hashingAlgorithm = "SHA-256"
     val keySize = 2048
-    val timeout = 60
+    val timeout = 5
```

Comment:
Even with time-based key, we sign it immediately, shorter the window to reduce the risk.